LMChatUI_iOS/LMChatUI_iOS/Source/UI/MessageList/LMChatMessageListView.swift

```diff
@@ -94,8 +94,9 @@ open class LMChatMessageListView: LMView {
             public var tempId: String?
             public var hideLeftProfileImage: Bool?
             public var pollData: LMChatPollView.ContentModel?
+            public var metadata: [String:Any]?
             
-            public init(messageId: String, memberTitle: String?, memberState: Int?, message: String?, timestamp: Int?, reactions: [Reaction]?, attachments: [Attachment]?, replied: [Message]?, isDeleted: Bool?, createdBy: String?, createdByImageUrl: String?, createdById: String?, isIncoming: Bool?, messageType: Int, createdTime: String?, ogTags: OgTags?, isEdited: Bool?, attachmentUploaded: Bool?, isShowMore: Bool, messageStatus: LMMessageStatus?, tempId: String?, hideLeftProfileImage: Bool?, pollData: LMChatPollView.ContentModel?) {
+            public init(messageId: String, memberTitle: String?, memberState: Int?, message: String?, timestamp: Int?, reactions: [Reaction]?, attachments: [Attachment]?, replied: [Message]?, isDeleted: Bool?, createdBy: String?, createdByImageUrl: String?, createdById: String?, isIncoming: Bool?, messageType: Int, createdTime: String?, ogTags: OgTags?, isEdited: Bool?, attachmentUploaded: Bool?, isShowMore: Bool, messageStatus: LMMessageStatus?, tempId: String?, hideLeftProfileImage: Bool?, pollData: LMChatPollView.ContentModel?, metadata: [String:Any]?) {
```

Comment:
add new line for params, it is not readable

Reply:
Fixed